include/base.php

```diff
@@ -41,6 +74,11 @@ public function __construct( &$links_model ) {
 	 * @since 2.6
 	 */
 	public function init() {
+		if ( $this->model->get_languages_list() ) {
```

Comment:
I have a problem with writing a condition in a method that is documented to 'instantiate the classes always needed'. I don't know what to change though (the method doc comment maybe).